docs/background/index.rst

```diff
@@ -27,11 +27,53 @@ TODO
 
 Hello World.
 
+.. _bg_models:
+
+Background Models
```

Comment:
I'd like the top-level `gammapy.background` page to remain short, just giving an overview of the available functionality and maybe one or two quick code examples / plots.

So please move this whole section to a sub page.
An example how to do this is here: https://gammapy.readthedocs.org/en/latest/image/plotting.html , i.e. plotting is a subpage of the image docs.